src/app/App.tsx

```diff
@@ -1,55 +1,30 @@
-import React, { useState } from "react";
-import logo from "./logo.svg";
+import React from "react";
 import styles from "./App.module.css";
-import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { BrowserRouter, Switch, Route, RouteProps } from "react-router-dom";
+import LandingPage from "./pages/LandingPage/LandingPage";
 
-function App(): JSX.Element {
-  const [count, setCount] = useState<number>(0);
+type CustomRouteProps = RouteProps & {
+  Component: () => JSX.Element;
+  path: string;
+};
+
+const routes: CustomRouteProps[] = [
+  { path: "/", Component: LandingPage, exact: true },
+];
 
+function App(): JSX.Element {
   return (
-    <Router>
-      <div className={styles.App}>
-        <header className={styles["App-header"]}>
-          <img src={logo} className={styles["App-logo"]} alt="logo" />
-          <p>Hello Vite + React!</p>
-          <p>
-            <button onClick={() => setCount((count) => count + 1)}>
-              count is: {count}
-            </button>
-          </p>
-          <p>
-            Edit <code>App.tsx</code> and save to test HMR updates.
-          </p>
-          <p>
-            <a
-              className={styles["App-link"]}
-              href="https://reactjs.org"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Learn React
-            </a>
-            {" | "}
-            <a
-              className={styles["App-link"]}
-              href="https://vitejs.dev/guide/features.html"
-              target="_blank"
-              rel="noopener noreferrer"
-            >
-              Vite Docs
-            </a>
-          </p>
-          <Switch>
-            <Route path="/help">
-              <p>Help</p>
-            </Route>
-            <Route path="/about">
-              <p>About</p>
+    <div className={styles.container}>
```

Comment:
What is the reason for `styles.container`?

Reply:
I renamed it to styles.App and gave it a max width now :) Thanks!